api/src/functions/health-check/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
```

Comment:
Let's try to get in the habit of defining the url path in here, rather than have the functions runtime imply it from the folder name.

Reply:
Speaking of which, how would you feel about changing this from 'health-check' to just 'health'? I like the semantics of "GET /api/health" more than "GET /api/health-check".

Reply:
I have changed the semantics to `GET /api/health` but the definition of URL doesn't seem to be available in the schema. In other words, we cannot define the URL for the function as it is assumed automatically from the name of the folder.

Sources -
1. https://github.com/Azure/azure-functions-host/wiki/function.json
2. http://json.schemastore.org/function

Marking this as resolved.